vwifi.c

```diff
@@ -1611,10 +1740,22 @@ static struct ieee80211_channel owl_supported_channels_2ghz[] = {
     CHAN_2GHZ(13, 2472), CHAN_2GHZ(14, 2484),
 };
 
-/* Array of supported rates, required to support at least those next rates
- * for 2GHz band.
+/* Array of "supported" channels in 5GHz band. It is required for wiphy. */
+static struct ieee80211_channel owl_supported_channels_5ghz[] = {
```

Comment:
Add `const` modifier.

Reply:
The ieee80211_supported_band structure in cfg80211 does not define the ieee80211_channel pointer as const. If we use const, it will cause compilation warnings and execution errors.

Reply:
How about defining `owl_supported_channels_5ghz` with `const` modifier, and use `kmemdup()` to copy `owl_supported_channels_5ghz` into `band->channels`? 

If do so, also add the `const` modifier on `ieee80211_supported_band`.

Reply:
But we cannot use `kmemdup()` globally unless `owl_supported_channels_5ghz[]` is declared inside `owcfg80211_add()`.

Reply:
Yes, we can statically define `ieee80211_channels[]` and `ieee80211_supported_band` globally with `const` modifier. And assign them to `wiphy` dynamically by `kmemdup()`.

Many drivers do so, you can check the brcmfmac for more details.

Reply:
In `brcmfmac`, it defines `ieee80211_channel` with `static`, then defines `ieee80211_supported_band` with `static const`, and finally assigns `ieee80211_supported_band` to `wiphy` using `kmemdup()` in `brcmf_setup_wiphy()`.
So should we modify it in this way?

Reply:
In my opinion, `ieee80211_channel` can be also defined as `static const`. I don't know why `brcmfmac` doesn't do that, but I think it's a good idea to do so.